CHANGELOG.md

```diff
@@ -22,6 +22,15 @@ All notable, unreleased changes to this project will be documented in this file.
 - Extend invoice object types with `Order` references - #11505 by @przlada
   - Add `Invoice.order` field
   - Add `InvoiceRequested.order`, `InvoiceDeleted.order` and `InvoiceSent.order` fields
+- Allow to mutate objects, by newly added `externalReference` field, instead of Saleor-assigned ID. Apply to following models: #11410 by @zedzior
```

Comment:
Is it related to this change?

Reply:
Just found the comment has wrong indent and looks like sub-comment, so I just took it out. However, looks like rebase restored it. Thanks!